app/build.gradle

```
@@ -89,16 +89,16 @@ dependencies {
     implementation 'org.jetbrains.kotlinx:kotlinx-serialization-json:1.4.1'
 
     // timber
-    implementation "com.jakewharton.timber:timber:5.0.1"
+    implementation 'com.jakewharton.timber:timber:5.0.1'
 
     //navigation
     implementation 'androidx.navigation:navigation-fragment-ktx:2.5.3'
     implementation 'androidx.navigation:navigation-ui-ktx:2.5.3'
 
     // CameraX
-    implementation "androidx.camera:camera-core:1.3.0-alpha02"
-    implementation "androidx.camera:camera-camera2:1.3.0-alpha02"
-    implementation "androidx.camera:camera-lifecycle:1.3.0-alpha02"
+    implementation 'androidx.camera:camera-core:1.3.0-alpha02'
+    implementation 'androidx.camera:camera-camera2:1.3.0-alpha02'
+    implementation 'androidx.camera:camera-lifecycle:1.3.0-alpha02'
```

```suggestion
    implementation 'androidx.camera:camera-core:1.2.0'
    implementation 'androidx.camera:camera-camera2:1.2.0''
    implementation 'androidx.camera:camera-lifecycle:1.2.0''
```

alpha 버전 사용할 이유 없으면 stable로 가는게 좋습니다.